addons/debugger/userscript.js

```diff
@@ -71,6 +70,14 @@ export default async function ({ addon, global, console, msg }) {
     // Don't scroll to blocks in the flyout
     if (block.workspace.isFlyout) return;
 
+    // Make sure the code tab is active
+    if (addon.tab.redux.state.scratchGui.editorTab.activeTabIndex !== 0) {
+      addon.tab.redux.dispatch({
+        type: "scratch-gui/navigation/ACTIVATE_TAB",
+        activeTabIndex: 0,
+      });
+    }
```

Comment:
Block won't flash here, console will error `Cannot read property 'blockDragSurface_' of null` when calling `getRelativeToSurfaceXY()`. Block should either flash properly (setTimeout of 0?) or we should not try to make it flash.